typedocReadMe.md

```diff
@@ -18,15 +46,92 @@ The study metadata defines elements such as the study title, authors, and descri
 The UI configuration tells reVISit how the UI should be laid out, such as which image to use for the study logo, whether to include a sidebar, the contact email, etc. For more detailed documentation on the UI configuration, check out the [documentation](/typedoc/interfaces/UIConfig.html).
 
 
-## Study Components
+## Components
+
+Components are the building blocks for each study. 
+There are currently 5 types of components: 
+* [Image](/typedoc/interfaces/ImageComponent.html), 
+* [Website](/typedoc/interfaces/WebsiteComponent.html), 
+* [Questionnaire](/typedoc/interfaces/QuestionnaireComponent.html), 
+* [React](/typedoc/interfaces/ReactComponent.html), and 
+* [Markdown](/typedoc/interfaces/MarkdownComponent.html).
+
+Each component has a list of responses which represents a set of questions to ask to the user for that particular component. The user can describe where the question should be displayed in the UI, the instruction for the response, and the type of response input (e.g., a [numerical response](/typedoc/interfaces/NumericalResponse.html), a [dropdown](/typedoc/interfaces/DropdownResponse.html), a [slider](/typedoc/interfaces/SliderResponse.html), etc). Each response interface extends the [BaseResponse](/typedoc/interfaces/BaseResponse.html) interface.
+
+
+### Collecting Responses
+
+Each component has a list of responses which represents a set of questions to ask to the user for that particular component. The user can describe where the question should be displayed in the UI, the type of response input (e.g., a [numerical response](/typedoc/interfaces/NumericalResponse.html), a [dropdown](/typedoc/interfaces/DropdownResponse.html), a [slider](/typedoc/interfaces/SliderResponse.html), and more. Each response interface extends the [BaseResponse](/typedoc/interfaces/BaseResponse.html) interface. 
```

Comment:
Duplicated blocks. Maybe we should just pick one or use the first sentence as a segue?

Reply:
Fixed in [4b1bc13](https://github.com/revisit-studies/study/pull/344/commits/4b1bc13396bb5e182335cd4e4c5637c67be01fd0)